newIDE/app/src/JsExtensionsLoader/BrowserJsExtensionsLoader.js

```diff
@@ -165,6 +165,12 @@ const jsExtensions = [
     extensionModule: require('GDJS-for-web-app-only/Runtime/Extensions/TextInput/JsExtension.js'),
     objectsRenderingServiceModules: {},
   },
+  {
+    name: 'Scene3D',
```

Comment:
3DScene?

Reply:
The extension name must start with a letter so that it can be recognized properly in expressions (if you write `3DScene::Something()`, the parser is confused because identifiers can't start with numbers, otherwise the grammar is ambiguous).